### PR TITLE
[civ2][data/2] move data arrow 6 + nightly + bulk executor tests to civ2

### PR DESCRIPTION
**Pull request by can-anyscale**
As title, move data arrow 6 + nightly tests to civ2

Test:
- CI

### CODE REVIEW

File: ci/docker/data6.build.wanda.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+srcs:
+  - ci/env/install-dependencies.sh
+  - python/requirements.txt
+  - python/requirements_compiled.txt
+  - python/requirements/test-requirements.txt
+  - python/requirements/ml/dl-cpu-requirements.txt
+  - python/requirements/ml/data-requirements.txt
+  - python/requirements/ml/data-test-requirements.txt
+build_args:
+  - ARROW_VERSION=6.*
```
Comment (aslonnie) [inline ci/docker/data6.build.wanda.yaml:13]: do these really need to be built 3 times?

where is `ARROW_VERSION` being used?

Comment (can-anyscale) [reply ci/docker/data6.build.wanda.yaml:13]: It's just a version of one single package https://github.com/ray-project/ray/blob/master/ci/env/install-dependencies.sh#L408; but yeah that simple thing cause 3 builds, unless the version can be switched at runtime

These 3 builds have a big shared layer but due to wanda caching I don't bother restructure this too much

Let me know if you have some other brilliant ideas!

Comment (aslonnie) [reply ci/docker/data6.build.wanda.yaml:13]: ok, let's use this for now, and maybe iterate later.

interesting that we just keep expanding our wanda bases..

**Review by aslonnie — APPROVED** [review]

File: ci/docker/data6.build.wanda.yaml (hunk 1 above)
Comment (can-anyscale) [reply ci/docker/data6.build.wanda.yaml:13]: I gonna blame the caching feature ;))

**Pull request merged**